sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/LiveMetrics/DataCollection/DocumentHelper.cs

```diff
@@ -118,42 +123,53 @@ internal static RemoteDependency ConvertToDependencyDocument(Activity activity)
+                    remoteDependencyDocument.Name = liveMetricsTagsProcessor.Tags.GetNewSchemaHttpDependencyName(httpUrl) ?? activity.DisplayName;
+
+                    remoteDependencyDocument.CommandName = httpUrl;
                     var httpResponseStatusCode = AzMonList.GetTagValue(ref liveMetricsTagsProcessor.Tags, SemanticConventions.AttributeHttpResponseStatusCode)?.ToString();
-                    remoteDependencyDocument.ResultCode = httpResponseStatusCode;
-                    remoteDependencyDocument.Duration = activity.Duration < SchemaConstants.RequestData_Duration_LessThanDays
-                                                ? activity.Duration.ToString("c", CultureInfo.InvariantCulture)
-                                                                : SchemaConstants.Duration_MaxValue;
+                    remoteDependencyDocument.ResultCode = httpResponseStatusCode ?? "0";
 
                     // The following "EXTENSION" properties are used to calculate metrics. These are not serialized.
                     remoteDependencyDocument.Extension_IsSuccess = IsHttpSuccess(activity, httpResponseStatusCode);
                     break;
                 case OperationType.Db:
-                    // Note: The Exception details are recorded in Activity.Events only if the configuration has opt-ed into this (SqlClientInstrumentationOptions.RecordException).
-
-                    var (_, dbTarget) = liveMetricsTagsProcessor.Tags.GetDbDependencyTargetAndName();
+                    remoteDependencyDocument.Name = activity.DisplayName;
 
-                    remoteDependencyDocument.Name = dbTarget;
                     remoteDependencyDocument.CommandName = AzMonList.GetTagValue(ref liveMetricsTagsProcessor.Tags, SemanticConventions.AttributeDbStatement)?.ToString();
-                    remoteDependencyDocument.Duration = activity.Duration.ToString("c", CultureInfo.InvariantCulture);
 
                     // TODO: remoteDependencyDocumentIngress.ResultCode = "";
                     // AI SDK reads a Number property from Connection or Command objects.
                     // As of Feb 2024, OpenTelemetry doesn't record this. This may change in the future when the semantic convention stabalizes.
 
-                    // The following "EXTENSION" properties are used to calculate metrics. These are not serialized.
-                    remoteDependencyDocument.Extension_IsSuccess = activity.Status != ActivityStatusCode.Error;
+                    var dbName = AzMonList.GetTagValue(ref liveMetricsTagsProcessor.Tags,, SemanticConventions.AttributeDbName)?.ToString();
+                    if (dbName != null)
+                    {
+                        remoteDependencyDocument.Properties.Add(new KeyValuePairString(SemanticConventions.AttributeDbName, dbName));
+                    }
                     break;
                 case OperationType.Rpc:
```

Comment:
Do we need to check for this case? In your previous PR RPC related check was removed from the exporter.

Reply:
Hi Raj, this PR is in Draft, you can ignore changes here for now.

Answering your question; No we don't need this. I think this section should be removed. 
I made these changes earlier in the week. Today I'm pushing my changes to save my progress.
I need to follow up with you offline about some different changes.